portfolio/src/main/webapp/index.html

```diff
@@ -1,39 +1,187 @@
+                <p style="font-style: italic;">a few of my favorite local eats</p>
+
+            </div>
+
+            <div class="row ">
+                <div class="caption-style-1">
+                    <div class="col-md-3 ">
+                        <div class="food-img ">
+                            <img style="width: 90%; height: 25%" src="images/food5.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Gourmonade <br>San Francisco, CA</p>
+                                </div>
+                            </div>		
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food6.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Two Hands <br>Little Italy, NY</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food3.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Home <br>San Francico, CA</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food4.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Ippodu <br>San Francico, CA</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>   
+                </div>
+            </div>
+
+            <div class="row food-second">
+                <div class="caption-style-1">
+                    <div class="col-md-3">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food1-new.jpg" alt="Anika's profile picture" />
```

Comment:
Alt text should be used to describe an image for users without sight if the description would be deemed useful.

An image of a floppy disk indicating a save icon may have

`alt="Save"`

In this case, I'm not sure these images need alt text (unless you'd like to write a description of the images) so instead I'd recommend

```
<img class="food-image" src="images/food1-new.jpg" alt>
```

Where specifying an empty alt-text will make sure nothing it said about the image, instead of the alt text defaulting to the image's URL.

```

Reply:
didn't catch this when I was copying - fixed it